jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/IteratingCallback.java

```diff
@@ -335,7 +338,13 @@ private void processing()
             finally
             {
                 if (callOnSuccess)
+                {
                     onSuccess();
+                    try (AutoLock ignored = _lock.lock())
+                    {
+                        processing = _state == State.PROCESSING;
+                    }
```

Comment:
Add an `isProcessing()` method as a companion to all the others, so this reads better.